Classes/Hooks/TypolinkHook.php

```diff
@@ -23,11 +24,25 @@
 class TypolinkHook
 {
     /**
+     * This Hook will convert typolinks to be used in Vue/Nuxt frontend,
+     * by modifying `lastTypoLinkUrl` with json encoded details about the previously generated typolink.
+     * If the static TypoScript Template won't be used, the default typolink behavior will be retained.
+     *
      * @param array $params
      * @param ContentObjectRenderer $ref
      */
     public function handleLink(array $params, ContentObjectRenderer $ref): void
     {
+
+        /** @var array $setup */
+        $setup = &$this->getTypoScriptFrontendController()->tmpl->setup;
+        if (! isset($setup['plugin.']['tx_headless.']['staticTemplate'])
+            || (bool)$setup['plugin.']['tx_headless.']['staticTemplate'] === false
```

Comment:
and add space between (bool) and $setup

Reply:
With the extra space between that PHPCS will fail, see https://github.com/TYPO3-Initiatives/headless/pull/143/checks?check_run_id=853587084
Should I then remove the space or should the code style definition for PHPCS be altered?

Reply:
you are right :) sorry